android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUISlider.java

```diff
@@ -91,11 +91,12 @@ public void processProperties(KrollDict d)
 		} else {
 			maxRange = max;
 		}
-		
 		if (d.containsKey("thumbImage")) {
 			updateThumb(seekBar, d);
 		}
-		
+		if (d.containsKey(TiC.PROPERTY_SPLIT_TRACK)) {
+			seekBar.setSplitTrack(TiConvert.toBoolean(d.get(TiC.PROPERTY_SPLIT_TRACK)));
```

Comment:
Good catch Yordan.

We also have to watch out in case the property value is not a boolean.  So, the code should look something like this...

```java
if (Build.VERSION.SDK_INT  >= 21) {
	ApiLevel21.setSplitTrack(this.seekBar, TiConvert.toBoolean([[value]], false));
}

private static class ApiLevel21
{
	private ApiLevel21() {}

	public void setSplitTrack(SeekBar seekBar, boolean enabled)
	{
		if (seekBar != null) {
			seekBar.setSplitTrack(enabled);
		}
	}
}
```

The static inner class above avoids the opcode warning on the lower API levels.